src/portals/hope/features/claims/claim-details/ClaimSubclaims/components/AddSubclaim.tsx

```diff
@@ -179,7 +180,10 @@ export const AddSubclaim: React.FC<{ claimId: string; memberId: string }> = ({
       case 2:
         return (
           <AddSubclaimStep>
-            {types.map((type: string, index) => {
+            {(selectedContract?.availableClaimTypes?.length
+              ? selectedContract.availableClaimTypes
+              : claimTypes
+            ).map((type: string, index) => {
```

Comment:
`availableClaimTypes` already defaults to `claimTypes` if the list is empty
```suggestion
            {availableClaimTypes).map((type: string, index) => {
```